go-controller/pkg/networkmanager/nad_controller.go

```diff
@@ -152,10 +164,66 @@ func (c *nadController) Start() error {
 
 func (c *nadController) Stop() {
 	klog.Infof("%s: shutting down", c.name)
+	close(c.stopChan)
 	controller.Stop(c.controller)
 	c.networkController.Stop()
 }
 
+func (c *nadController) Reconcile(key string) {
+	c.controller.Reconcile(key)
+}
+
+// ForceReconcile enqueues a sync for a given NAD.
+// - For local NADs, it may start or cancel grace-period cleanup.
+// - For remote NADs, it triggers immediate reconcile but never affects cleanup timers.
```

Comment:
Do you mean to say local and remote node or is it remote and local NAD? because I don't know what remote NAD means